src/main/java/com/example/backendspringboottechiteasycontroller/controllers/TelevisionController.java

```diff
@@ -0,0 +1,53 @@
+package com.example.backendspringboottechiteasycontroller.controllers;
+
+import com.example.backendspringboottechiteasycontroller.models.Television;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+@RestController
+public class TelevisionController {
+
+    List<Television> televisionDatabase = new ArrayList();
```

Comment:
This needs to be the repository now

Reply:
In the TechItEasy-service/dto exercise, we won't be using the repository in the controller, but rather in the service and in the service we'll be using the repository.